ocaml/xapi/extauth_ad.mli

```diff
@@ -0,0 +1,11 @@
+(* start AD external auth backend daemon *)
```

Comment:
We also need a copyright header on this new file.

Reply:
oh, sorry, I have missed this comment,
I will add a license in the next commit, sorry about this.